### PR TITLE
fix(templates): fix content creation using template

### PR DESCRIPTION
**Pull request by petrjasek**
- remove template if user saves no changes after creating it.
- use also default desk when creating it so it won't end up
in user personal space.

SD-3431

### CODE REVIEW

File: client/app/scripts/superdesk-workspace/content/content.spec.js [hunk 1]
```diff
@@ -71,7 +71,8 @@ describe('superdesk.workspace.content', function() {
                 slugline: 'test_slugline',
                 body_html: 'test_body_html',
                 task: {desk: '2', stage: '4', user: 'user:1'},
-                template: 'template1'
+                template: 'template1',
+                version: 0
```
Comment (akintolga) [inline client/app/scripts/superdesk-workspace/content/content.spec.js]: There's a change in requirements. Lasky must have updated the ticket with the change. In brief, the brand new items will be saved as v0 but the items created with some data in them suggested to be v1:
* Packages created by selecting an item
* Updates of published stories
* Stories out of templates

Please take a look at this PR as well:
https://github.com/superdesk/superdesk/pull/1602

Comment (mdhaman) [inline client/app/scripts/superdesk-workspace/content/content.spec.js]: And `Takes` as well.

Comment (petrjasek) [inline client/app/scripts/superdesk-workspace/content/content.spec.js]: ok this is not affecting items created by system so it should work as you said

**Comment by ioanpocol** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**